paddle/fluid/framework/details/build_strategy.cc

```diff
@@ -53,6 +54,8 @@ class ParallelExecutorPassBuilder : public ir::PassBuilder {
                         "sequential_execution_pass");
     AppendPassWithCheck(strategy_.sync_batch_norm_, "sync_batch_norm_pass");
 
+    AppendPassToUseNgraph("ngraph_subgraph_pass");
```

Comment:
add "if FLAGS_use_ngraph" to make this line only works for ngraph?
otherwise, there may have a error if ngraph is not defined and used.